src/agenteval/leaderboard/dataset_features.yml

```diff
@@ -17,7 +17,7 @@
       - name: primary_metric
         dtype: string
       - name: tags
-        sequence: string
```

Comment:
Don't think this is related, but CI failure asked me to update the schema.

Reply:
Hmm. I don't think the underlying data structure has changed so I'm wondering if it's a library change (pydantic, e.g.). I fear chaos, but maybe it's fine.

Reply:
@jbragg Do you have a sense why CI might have failed? I think Stefan just made this change trying to make CI pass, but no schema changes should be needed.